code/iaas/agent-instance/src/main/java/io/cattle/platform/agent/instance/service/impl/AgentInstanceManagerImpl.java

```diff
@@ -64,9 +72,8 @@
                         .forVnetId(nic.getVnetId())
                         .withSystemContainerType(SystemContainer.NetworkAgent)
                         .build();
-            } else {
-                start(agentInstance);
             }
+            start(agentInstance);
```

Comment:
One more leftover from the thing I've tried to implement, and that didn't work with idempotent checks on so I rolled changes back. I've tried to implement creating system Instance w/o starting it in AgentInstanceFactory, and delegating the start to AgentInstanceManager/LoadBalancerManager instead. Will rollback this as well.